app/templates/components/public/ticket-list.hbs

```diff
@@ -47,7 +50,13 @@
               </td>
             {{else}}
               <td id="{{ticket.id}}_price">
-                {{currency-symbol eventCurrency}} {{format-number ticket.price}}
+                {{#if (eq ticket.type 'donation') }}
+                  <div class="field {{if (eq (donation-valid ticket.price ticket.minPrice ticket.maxPrice) false) 'error'}}" >
```

Comment:
Validation for the public page can still be achieved with semantic-ui validator. So why have you not used it ?

Reply:
@niranjan94 Yeah, Will do that.

Reply:
@niranjan94 The validation for it being between the minimum and maximum price wasn't working. For it to be greater than 1(`integer[1..]`), it's working. `checkMinMax[max_price, min_price]` doesn't seem to work.

Reply:
@niranjan94 What do you suggest?

Reply:
@mrsaicharan1 what is stopping you from using `integer` ? That can be used easily for this right ?

Generate the validation rules dynamically ... One rule per ticket with varying integer ranges based on that ticket's min/max price

Reply:
@niranjan94 Done.

Reply:
> Generated one rule per ticket and merged it with the validation rules

Perfect 🎉